### DISCUSSION TITLE
Knime RDKit nodes "RDKit Canon Smiles" and "RDKit to InChI" are crashing Knime 4.4.4

### DISCUSSION THREAD

**Discussion by ddsneo4j** [rdkit/rdkit#5227] [Development]
Hi,

  unfortunately, the RDKit nodes "RDKit Canon Smiles" and "RDKit to InChI" are crashing Knime 4.4.4 - see attached Knime workflow and input structure. Could this bug please be fixed, i.e. Knime should not be crashed by these nodes because of a structure where canonical smiles and InChI keys cannot be created for?

Thanks for your effort in advance.

Best regards,
Dan
[RdKit.zip](https://github.com/rdkit/rdkit/files/8539932/RdKit.zip)

**Comment by greglandrum** [comment]
This is a topic for the knime-rdkit repo (https://github.com/rdkit/knime-rdkit/issues), not this one.
